charts/turing/Chart.yaml

```diff
@@ -22,4 +22,4 @@ maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
 name: turing
-version: 0.3.3
+version: 0.3.4
```

Comment:
Should we also bump the `appVersion` to `1.16.0`? https://github.com/caraml-dev/turing/releases/tag/v1.16.0
(Can't seem to leave a comment on an unchanged line on GitHub :/)